packages/scandipwa/src/component/Router/Router.component.tsx

```diff
@@ -311,6 +311,19 @@ export class RouterComponent extends PureComponent<RouterComponentProps, RouterC
         });
     }
 
+    setDemoNoticeHeight(): void {
+        const {
+            storeConfig: {
+                demo_notice,
+            } = {},
+        } = window;
+
+        if (demo_notice) {
+            const root = document.querySelector(':root') as HTMLElement;
+            root?.style.setProperty('--demo-notice-height', '48px');
```

Comment:
Same as 48px should be a some constant, otherwise it is not know where comes from and what to change